tests/garage/sampler/test_ray_sampler.py

```diff
@@ -0,0 +1,49 @@
+import gym
+import numpy as np
+import ray
+
+from garage.sampler import RaySampler, SamplerWorker
+from garage.tf.envs import TfEnv
+from garage.tf.samplers.on_policy_vectorized_sampler \
+    import OnPolicyVectorizedSampler
+from tests.fixtures.policies.ray_sampler_utils import MockAlgo, ScriptedPolicy
+
+
+class TestSampler():
+    @staticmethod
+    def test_ray_batch_sampler():
+        ray.init(
+            ignore_reinit_error=True,
+            object_store_memory=10000000,
+            redis_max_memory=10000000)
+        gym.envs.register(
+            id='GridWorldEnv-v0',
+            entry_point='garage.envs:GridWorldEnv',
+            max_episode_steps=16,
+            kwargs={'desc': '4x4'},
+        )
+
+        env = TfEnv(gym.make('GridWorldEnv-v0'))
+
+        policy = ScriptedPolicy()
+
+        algo = MockAlgo(env.spec, policy)
+        sampler1 = RaySampler(
+            algo, env, num_processors=1, sampler_worker_cls=SamplerWorker)
+        sampler1.start_worker()
+
+        sampler2 = OnPolicyVectorizedSampler(algo, env)
```

Comment:
this is a nice test to verify what you've done is working, but to commit this to master you need to add unit tests which cover your classes individually, instead of just comparing the output of a single sampling process to the current sampler. see tests/garage/tf/models for some good unit tests.